libs/knowledge-store/README.md

```diff
@@ -22,7 +22,8 @@ The Knowledge Store makes use of the following metadata fields on each `Document
 To connect nodes based on hyperlinks, you can use the `HtmlLinkEdgeExtractor` as shown below:
 
 ```python
-from ragstack_knowledge_store.html_link_edge_extractor import HtmlLinkEdgeExtractor
+from ragstack_knowledge_store.extractors.html_link_edge_extractor import
```

Comment:
Could `extractors` export this, so we can do `from ragstack_knowledge_store.extractors import HtmlLinkEdgeExtractor`?

Reply:
done